packages/app/src/domain/topical/components/indicator-level-description.tsx

```diff
@@ -0,0 +1,35 @@
+import { SeverityIndicatorLevel } from '~/components/severity-indicator-tile/components/severity-indicator-label';
+import { SeverityLevel } from '~/components/severity-indicator-tile/types';
+import { replaceVariablesInText } from '~/utils';
+import { asResponsiveArray } from '~/style/utils';
+import { Text } from '~/components/typography';
+import { Box } from '~/components/base';
+import css from '@styled-system/css';
+import { space } from '~/style/theme';
+
+interface indicatorLevelDescriptionProps {
```

Comment:
Type and interface definitions always `PascalCase`
```suggestion
interface IndicatorLevelDescriptionProps {
```